ios/RN/RNCamera.m

```diff
@@ -351,6 +357,57 @@ - (void)updateWhiteBalance
     [device unlockForConfiguration];
 }
 
+
+/// Set the AVCaptureDevice's ISO values based on RNCamera's 'exposure' value,
+/// which is a float between 0 and 1 if defined by the user or -1 to indicate that no
+/// selection is active. 'exposure' gets mapped to a valid ISO value between the
+/// device's min/max-range of ISO-values.
+///
+/// The exposure gets reset every time the user manually sets the autofocus-point in
+/// 'updateAutoFocusPointOfInterest' automatically. Currently no explicit event is fired.
+/// This leads to two 'exposure'-states: one here and one in the component, which is
+/// fine. 'exposure' here gets only synced if 'exposure' on the js-side changes. You
+/// can manually keep the state in sync by setting 'exposure' in your React-state
+/// everytime the js-updateAutoFocusPointOfInterest-function gets called.
+- (void)updateExposure
+{
+    AVCaptureDevice *device = [self.videoCaptureDeviceInput device];
+    NSError *error = nil;
+    
+    if (![device lockForConfiguration:&error]) {
+        if (error) {
+            RCTLogError(@"%s: %@", __func__, error);
+        }
+        return;
+    }
+    
+    // Check that either no explicit exposure-val has been set yet
+    // or that it has been reset. Check for > 1 is only a guard.
+    if(self.exposure < 0 || self.exposure > 1){
+        [device setExposureMode:AVCaptureExposureModeContinuousAutoExposure];
+        [device unlockForConfiguration];
+        return;
+    }
+    
+    // Lazy init of range.
+    if(!self.exposureIsoMin){ self.exposureIsoMin = device.activeFormat.minISO; }
+    if(!self.exposureIsoMax){ self.exposureIsoMax = device.activeFormat.maxISO; }
+    
```

Comment:
Why do we set `exposureIsoMin` and `exposureIsoMax` to a global variable? We only ever use it on line 399. Is it expesnive to access `device.activeFormat.[min/max]ISO`?

Reply:
My thought was that by storing the min/max values we don't need access to the device-var at a later point in time.

Reply:
I don't think it's an expensive operation. It might be preferable to keep globals to a minimum. But this is not a blocking comment. Great job man.